src/index.js

```diff
@@ -0,0 +1,113 @@
+const MAX_TIMEOUT_ATTEMPTS = 3;
+const MAX_TIMEOUT = 1000;
+
+
+function main(client) {
+	let timeoutAttempts = 0;
+	let appliedClasses = [];
+	let unappliedFunctions = new Set();
+	let appliedFunctions = new Set();
+
+	function retrieveEvolvCssAsset() {
+		const styleSheets = document.getElementsByTagName("link");
+
+		for (let i = 0; i < styleSheets.length; i++) {
+			const link = styleSheets[i];
+			if (link.href &&
+					link.href.indexOf('evolv.ai') >= 0 &&
+					link.href.indexOf('assets.css') >= 0) {
+				return link;
+			}
+		}
+
+		return undefined;
+	}
+
+	function retrieveEvolvJsAsset() {
+		const scripts = document.scripts;
+	
+		for (let i = 0; i < scripts.length; i++) {
+			const script = scripts[i];
+			if (script.src && script.src.indexOf('evolv.ai') >= 0 && script.src.indexOf('assets.js') >= 0) {
+				return script;
+			}
+		}
+	
+		return undefined;
+	}
+
+	const invokeFunctions = function () {
+		const evolv = window.evolv;
+		if (typeof evolv === 'undefined' || !evolv.javascript || !evolv.javascript.variants) {
+			if (timeoutAttempts < MAX_TIMEOUT_ATTEMPTS) {
+				setTimeout(this, MAX_TIMEOUT);
```

Comment:
Using `this` may work, but it is non-standard. This is more clear.
```suggestion
				setTimeout(invokeFunctions, MAX_TIMEOUT);
```